doc/api-extensions.md

```diff
@@ -757,3 +757,6 @@ migration is required.
 If the kernel supports seccomp-based syscall interception LXD can be notified
 by a container that a registered syscall has been performed. LXD can then
 decide to trigger various actions.
+
+## container\_nic\_ipvlan
```

Comment:
This should be in a separate `api: IPVLAN support` commit along with adding the string to `shared/version/api.go`.